guardian/master-to-main
# fix: Revert conversion to ESM

## What does this change?

This reverts the conversion of the repo to use ESM instead of CommonJS. Unfortunately the released ESM package does not run. While troubleshooting has revealed that this is due to the `m2m` command not pointing to the correct run file, when it is corrected, there are other errors upon running the program. 

This has proved too time-consuming to pursue troubleshooting at the moment, so this reversion will allow a working version of the package to be released. 

## How to test

Using `yarn pack` will create a local tar file which you can install globally. Running `m2m` with any of its flags and/or arguments in the terminal should indicate that the package is running successfully.

## How can we measure success?

Users will be able to download the latest version of the package from NPM and it will run.

## Have we considered potential risks?

Not converting to ESM means that two libraries, Chalk and Ora, cannot be bumped beyond certain versions - this could have implications for security if there were vulnerabilities to be found in those packages' versions in the future. However, this repo has a limited useful lifetime as the [Github tools to convert repos get better](https://github.com/github/renaming#renaming-existing-branches).
